Open call to new champions program cohort 2025 (#983)
ropensci/roweb3: content/blog/2025-03-10-open-call-champions/index.es.md

---
slug: campeones-america-latina
title: 'Nueva cohorte del Programa de Campeon(a|e)s rOpenSci'
author: Yanina Bellini Saibene

> editor: Steffi LaZerte

Author: ```suggestion
editor: 
- Steffi LaZerte
- Alejandra Bellini
```

## ¿Por qué ser mentor/a?

Desempeñarás un papel fundamental en el programa de rOpenSci, ayudando a seleccionar, asesorar e inspirar a quienes participen como campeones. 

Pondrás en contacto a campeones/as con personas, programas y organizaciones; recomendarás recursos, lecturas, formación y experiencias; y proporcionarás comentarios sobre el proyecto que desarrollen.

Recibirás entrenamiento sobre mentoría, adquirirás experiencia y habilidades para realizar tu tarea, un estipendio anual por tu participación, se te reconocerá en el sitio web de rOpenSci y en otros canales de comunicación como las redes sociales, y podrás hacer nuevas conexiones a través de las redes de las otras personas que participan.

Con tu mentoria y ayuda para dar forma a este programa, también estarás retribuyendo a la comunidad rOpenSci de una forma estupenda y significativa.

## ¿Cómo aplicar?

Se debe completar el formulario de aplicación antes del **miércoles 30 de abril de 2025**. Los formularios deben completarse en **español**. 

Si quieren conocer la información necesaria para completar el formulario [pueden verla en la web del programa](https://ropenscilabs.github.io/ChampionsProgram/). 

> * Formulario para aplicar como Mentor(a|e)s: 

Author: ```suggestion
* [Completa el formulario para aplicar como Mentor(a|e)s](https://airtable.com/appF6OXmxkk8VmR8a/shrZnWDRKxXUEmLRM) 
```

> * Formulario para aplicar como Campeon(a|e)s:

Author: ```suggestion
* [Completa el formulario para aplicar como Campeon(a|e)s](https://airtable.com/appF6OXmxkk8VmR8a/shrlvgcNz1R4CoaQN) 
```

Para las personas que quieran ser mentoras, buscamos que tengan experiencia previa desarrollando, manteniendo o contribuyendo a paquetes de R (si son de rOpenSci es aún mejor), o que hayan participado en el proceso de revisión de rOpenSci o en otro proyecto de revisión de código por pares. También es importante que estén dispuestas a guiar y enseñar a otras personas y a trabajar en colaboración. 

Pueden ver más detalles sobre los requisitos para participar en cada rol en [nuestro sitio web especifico del programa (en español)](https://ropenscilabs.github.io/ChampionsProgram/).

## ¿Por qué ser campeon/a?

A lo largo de 12 meses, recibirás formación, tutoría individual para el desarrollo de un proyecto de software libre, un estipendio anual y participarás en diversas actividades de construcción y fortalecimiento de comunidades.

Tambien debes desarrollar un proyecto como la creación de nuevos paquetes R o la participación en el proceso de revisión por pares de rOpenSci que enseña las mejores prácticas para desarrollar y compartir software de investigación y te invita a formar parte de una comunidad internacional.

> Al final del programa, los participantes pueden elegir cómo quieren retribuir a sus propias comunidades e instituciones, ya sea a través de la divulgación, la tutoría o la promoción de prácticas de ciencia abierta y software libre de investigacion.

Author: ```suggestion
Al final del programa,  puedes elegir cómo quieres retribuir a tus propias comunidades e instituciones, ya sea a través de la divulgación, la tutoría o la promoción de prácticas de ciencia abierta y software libre de investigacion.
```

> Cuando terminan, están plenamente integrados en la comunidad de rOpenSci, que les proporciona apoyo continuo y oportunidades de crecimiento.

Author: ```suggestion
Cuando termines el programa, estarás plenamente integrado en la comunidad de rOpenSci, que te proporciona apoyo continuo y oportunidades de crecimiento.
```

> ## ¿Por qué ser un mentor/a?

Author: ```suggestion
## ¿Por qué ser mentor/a?
```

> Mentores y mentoras desempeñarán un papel fundamental en el programa de rOpenSci, ayudando a seleccionar, asesorar e inspirar a quienes participen como campeones. 

Author: ```suggestion
Desempeñarás un papel fundamental en el programa de rOpenSci, ayudando a seleccionar, asesorar e inspirar a quienes participen como campeones. 
```

> Los mentores y mentoras pondrán en contacto a sus campeones/as con personas, programas y organizaciones; recomendarán recursos, lecturas, formación y experiencias; y proporcionarán comentarios sobre el proyecto que desarrollen.

Author: ```suggestion
Pondrás en contacto a campeones/as con personas, programas y organizaciones; recomendarás recursos, lecturas, formación y experiencias; y proporcionarás comentarios sobre el proyecto que desarrollen.
```

> Buscamos personas que cuenten con experiencia previa en desarrollar, mantener o contribuir a paquetes de R (si son de rOpenSci aún mejor), o haber participado en el proceso de revisión de rOpenSci u otro proyecto de revisió de código entre pares.

Author: ```suggestion

```

> Los mentores y mentoras tendran entrenamiento sobre mentorias, adquirirán experiencia y habilidades para realizar su tarea, un estipendio anual por su participación, serán reconocidos en el sitio web de rOpenSci y en los canales de las redes sociales, y harán nuevas conexiones a través de las redes de las otras personas que particpan.

Author: ```suggestion
Recibirás entrenamiento sobre mentoría, adquirirás experiencia y habilidades para realizar tu tarea, un estipendio anual por tu participación, se te reconocerá en el sitio web de rOpenSci y en otros canales de comunicación como las redes sociales, y podrás hacer nuevas conexiones a través de las redes de las otras personas que participan.
```

> Al tutelar a nuestros campeones y campeonas seleccionados y ayudar a dar forma a este programa, los tutores y tutoras también estarán retribuyendo a la comunidad rOpenSci de una forma estupenda y significativa.

Author: ```suggestion
Con tu mentoria y ayuda para dar forma a este programa, también estarás retribuyendo a la comunidad rOpenSci de una forma estupenda y significativa.
```

## ¿Cómo aplicar?

> Se debe completar el formulario de aplicación antes del **miércoles 30 de abril de 2025**. Los formularios deben completarse en español. 

Author: ```suggestion
Se debe completar el formulario de aplicación antes del **miércoles 30 de abril de 2025**. Los formularios deben completarse en **español**. 
```

Si quieren conocer la información necesaria para completar el formulario [pueden verla en la web del programa](https://ropenscilabs.github.io/ChampionsProgram/). 

> * Formulario para aplicar como Mentor(a|e) s: 

Author: ```suggestion
* Formulario para aplicar como Mentor(a|e)s: 
```

---
slug: campeones-america-latina
title: 'Nueva cohorte del Programa de Campeon(a|e)s rOpenSci'
author: Yanina Bellini Saibene
editor: 
- Steffi LaZerte
- Alejandra Bellini
date: '2025-03-10'

> tags:

Author: ```suggestion
featured: true
tags:
```

- campeones
- multilingüe
description: "Se abre el llamado para una nueva cohorte de nuestro Programa de Campeon(a|e)s con foco en América Latina. ¡Postúlate!"
params:

>   doi: ""

Author: ```suggestion
  doi: "10.59350/gvg6h-24382"
```

---

> ¡Tenemos grandes noticias! ¡Ya esta abierta la convocatoria para ser parte de la nueva cohorte de nuestro Programa durante 2025! ¡Y por primera vez será en español!

Author: ```suggestion
Tenemos grandes noticias: ¡Ya esta abierta la convocatoria para ser parte de la nueva cohorte de nuestro Programa durante 2025! ¡Y por primera vez será en español!
```

Nuestro programa busca identificar, reconocer y recompensar a personas que son líderes en una comunidad de ciencia abierta, ingeniería de software de investigación y la comunidad de programación R.  

El programa de este año está enfocado en personas de América Latina y por primera vez se llevará a cabo completamente en español.  El objetivo principal es fomentar el software de investigación sostenible como pilar de la Ciencia Abierta en América Latina mediante el desarrollo de capacidades y comunidad.

El programa es una plataforma increíble para desarrollar tu proyecto de código abierto con el apoyo de personas expertas, interactuar, compartir y fortalecer una red con otras personas decididas a desarrollar ciencia abierta y reproducible en tu parte del mundo. 

> La convocatoria ya está abierta para postularte como campeón/a o mentor/a y ser parte de la comunidad global de personas que usan y desarrollan software científico y de código abierto.

Author: ```suggestion

```

## ¿Quiénes pueden participar?

Buscamos personas que trabajen en STEAM, que sean de América Latina, que tengan un buen manejo oral y escrito de la lengua española y que ya tengan conocimiento y experiencia desarrollando con R. 

> Para quienes quieran ser mentores, además, es importante que tengan ganas de guiar y enseñar a otras personas y trabajar colaborativamente. 

Author: ```suggestion
Para las personas que quieran ser mentoras, buscamos que tengan experiencia previa desarrollando, manteniendo o contribuyendo a paquetes de R (si son de rOpenSci es aún mejor), o que hayan participado en el proceso de revisión de rOpenSci o en otro proyecto de revisión de código por pares. También es importante que estén dispuestas a guiar y enseñar a otras personas y a trabajar en colaboración. 
```

Con tu mentoria y ayuda para dar forma a este programa, también estarás retribuyendo a la comunidad rOpenSci de una forma estupenda y significativa.

## ¿Cómo aplicar?

Se debe completar el formulario de aplicación antes del **miércoles 30 de abril de 2025**. Los formularios deben completarse en **español**. 

Si quieren conocer la información necesaria para completar el formulario [pueden verla en la web del programa](https://ropenscilabs.github.io/ChampionsProgram/). 

* [Completa el formulario para aplicar como Mentor(a|e)s](https://airtable.com/appF6OXmxkk8VmR8a/shrZnWDRKxXUEmLRM) 

* [Completa el formulario para aplicar como Campeon(a|e)s](https://airtable.com/appF6OXmxkk8VmR8a/shrlvgcNz1R4CoaQN) 

Si tenes más dudas o preguntas podes participar de la [_Community Call_ que se realizará el 12 de marzo de 2025](/es/commcalls/champions-latino-2025/) donde participarán mentores y campeones de cohortes anteriores contando sus experiencias en el programa.

> Además, tendremos una [clínica de aplicación]() el **martes 15 de Abril de 2025**. En esta clínica, la coordinadora del programa estará disponible para responder preguntas y ayudar a los postulantes a completar el formulario de aplicación.

Author: ```suggestion
Además, tendremos una [clínica de aplicación](/events/clinica-champions-2025-04/) el **martes 15 de Abril de 2025**. En esta clínica, la coordinadora del programa estará disponible para responder preguntas y ayudar a los postulantes a completar el formulario de aplicación.
```
